java/demo/src/main/java/com/google/demo/template/ResultErrorTemplates.java

```diff
@@ -0,0 +1,146 @@
+// This file was automatically generated by the Soy compiler.
+// Please don't edit this file by hand.
+// source:
+// /Users/rogg/IdeaProjects/libphonenumber/java/demo/src/main/resources/com/google/demo/result_error.soy
```

Comment:
Manual. https://github.com/google/closure-templates/blob/master/documentation/codelabs/helloworlds/helloworld_java.md#using-soyparseinfogenerator
I removed the local paths

Reply:
Can we tie to the build process? As per the link you sent 

> Typically, you would want to run SoyParseInfoGenerator as part of a build step. This can be done with exec-maven-plugin for example, but it is outside the scope of this codelab.

https://www.mojohaus.org/exec-maven-plugin/exec-mojo.html

Reply:
Done